services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/network/PathComputationSpec.groovy

```diff
@@ -0,0 +1,108 @@
+package org.openkilda.functionaltests.spec.network
+
+import org.openkilda.functionaltests.HealthCheckSpecification
+import org.openkilda.functionaltests.extension.failfast.Tidy
+import org.openkilda.messaging.model.system.KildaConfigurationDto
+import org.openkilda.model.PathComputationStrategy
+import org.openkilda.testing.model.topology.TopologyDefinition.Isl
+
+class PathComputationSpec extends HealthCheckSpecification {
+
+    @Tidy
+    def "Default path computation strategy is used when flow does not specify it"() {
+        given: "Default path computation strategy is COST"
+        def initConfig = northbound.getKildaConfiguration()
+        northbound.updateKildaConfiguration(
+                new KildaConfigurationDto(pathComputationStrategy: PathComputationStrategy.COST.toString()))
```

Comment:
I don't think that this is necessary, but it's clean and explicit, IDEA inspector would warn me otherwise. Groovy may be able to handle that, have to check, but without `toString()` you will be trying to put `PathComputationStrategy` type into `String` field

Reply:
Removed toString()